securedrop_client/gui/__init__.py

```diff
@@ -195,3 +196,43 @@ def get_elided_text(self, full_text: str) -> str:
 
     def is_elided(self) -> bool:
         return self.elided
+
+
+class SecureQPlainTextEdit(QPlainTextEdit):
+    MAX_TEXT_WIDTH = 75
+    MAX_NATURAL_TEXT_WIDTH = 650
+    HEIGHT_BASE = 60
+    LINE_HEIGHT = 20
```

Comment:
so I'm currently investigating this height setting - how was this `LINE_HEIGHT` constant determined? (asking because I was wondering if we can get this on the fly, looks like it's not exposed on `QFont`)

Reply:
Though we do have `QFontMetrics::lineSpacing` and `QFontMetrics::height` which both report 16: 

```
$ fm = QFontMetrics(self.document().defaultFont())
$ print('text height', fm.height())
text height 16
$ print('text line spacing', fm.lineSpacing())
text line spacing 16
```

Reply:
> so I'm currently investigating this height setting - how was this `LINE_HEIGHT` constant determined? (asking because I was wondering if we can get this on the fly, looks like it's not exposed on `QFont`)

@creviera ^^